org.openstenoproject.Plover.metainfo.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<component type="desktop-application">
+  <id>org.openstenoproject.Plover</id>
+  <name>Plover</name>
+  <summary>Free and open source stenography engine</summary>
+  <developer id="dev.zed">
+    <name translate="no">Open Steno Project</name>
+  </developer>
+  <description>
+    <p>
+      <b
+      >NOTE:</b> This is the official release of Plover, packaged into a Flatpak by the community. This repackaging is not yet supported by the Open Steno Project.
+    </p>
+    <p>
+      Before there was Plover, there was deposition software, often called computer-aided transcription (CAT) software. While this software is very useful for stenographers making depositions, it is not well-fitted to other purposes. It is sandboxed, proprietary, and often bloated. Plover is a small, slick Python application that you run in the background. It acts as a translator to read steno input from your keyboard or steno machine and then emulates keystrokes. The programs you use can't tell that you're using steno.
+    </p>
+    <p>
+      This means that you can use Plover to chat on Facebook, write in Microsoft Word, browse the web, control your media, use a terminal, fire off keyboard shortcuts, open and close programs, navigate with arrow keys, write code, or anything else you could do with a regular keyboard, but at potentially much greater speeds!
+    </p>
+    <p>
+      Plover is used today by over a thousand people, of varying skills. It is used by both professional stenographers, transcriptionists, programmers, and hobbyists (for general computer use and composition). If you want to know more about the community, it's best to join our <a
+        href="https://discord.gg/0lQde43a6dGmAMp2"
```

Comment:
this is not HTML.

Reply:
it is in fact, valid HTML 😅, you probably meant that it's not valid appstream XML...